### DISCUSSION TITLE
Live-updating tables

### DISCUSSION THREAD

**Discussion by zackkatz** [UseDataKit/SDK#12] [Ideas]
It would be great if, when submitting a form, you can see the entry appear in the table.

This can be done in different ways. Some of them include:

- Add a repeating poll to check whether the cache has been busted. If it has, update the values.
- Add a listener for forms that match the same IDs (would have to be a method implemented differently for each data source).
- [Server-sent events](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events) could receive new submissions for a DataView

**Comment by doekenorg** [comment]
SSE is very server dependent. It could be an option though, but long polling should probably be the default, to prevent days sources from implementing another method. SSE is the nicest solution though. 👌
